css/style.css

```diff
@@ -150,40 +112,17 @@ div.autocomplete-deluxe-container input.autocomplete-deluxe-form {
 }
 
 .gin--edit-form .page-wrapper__node-edit-form .layout-region-node-secondary .gin-sidebar .form-actions .form-submit {
-  margin-top: 0;
   font-size: 1rem;
   max-height: 48px;
 }
-.gin--edit-form .page-wrapper__node-edit-form .layout-region-node-secondary .gin-sidebar .form-actions .form-submit:hover {
-  background-color: #0550e6;
-  color: white;
-}
+
 .gin--edit-form .page-wrapper__node-edit-form .layout-region-node-secondary .gin-sidebar .form-actions .save-edit .button:hover {
   background-color: #0e716c;
+  color: white;
 }
 
-.page-node-type-article .layout-region-node-actions {
-  margin-right: 150px;
-}
-
-
-@media screen and (max-width: 1024px) {
-  .page-node-type-article .layout-region-node-actions {
-    margin-bottom: 60px;
-    display: block;
-  }
-
-  .layout-region-node-actions .form-actions {
-    display: block;
-  }
-
-  .layout-region-node-actions .form-actions input {
-    display: block;
-  }
-}
-
-.layout-region-node-actions {
-  margin-right: 150px;
+.js .paragraphs-add-dialog--categorized .paragraphs-add-dialog-row button {
+    max-width: 212px!important;
```

Comment:
Please elaborate on why we need `!important` here

Reply:
in add add paragraph content model box , we have to show three category in a row. for do that this code implement, _same line of code used in gin theme, so css not implement and parent css class is available for that. that's way **!important** is used here_

Reply:
Please add this reason as a comment in the code, for future reference.

Reply:
Done